extensions/kubernetes/spi/src/main/java/io/quarkus/kubernetes/spi/ConfiguratorBuildItem.java

```diff
@@ -1,11 +1,9 @@
 package io.quarkus.kubernetes.spi;
 
-import java.util.Optional;
-
 import io.quarkus.builder.item.MultiBuildItem;
 
 /**
- * A build item that wraps around Configurator objects.
+ * A build item that wraps around Configurator or ConfigurationSupplier objects.
```

Comment:
@iocanel wdyt about this change? Initially, I was going to create another build item to wrap `ConfigurationSupplier` only, but taking into account that the ConfiguratorBuildItem and the ConfigurationSupplierBuildItem deal both with the ConfigurationRegistry of dekorate (same `add` method), I decided to reuse the ConfiguratorBuildItem for ConfigurationSupplier and Configurator both. But let me know if you want me to change it.

Reply:
Given that we try to avoid referring to specific dekorate types in the spi, it might make it harder to distinguish between the two.

Reply:
Separate into a new build item